components/Account/Access/Campaigns/index.js

```diff
@@ -0,0 +1,75 @@
+import { compose, graphql } from 'react-apollo'
+import gql from 'graphql-tag'
+
+import Campaign from './Campaign'
+
+import query from '../../belongingsQuery'
+
+const Campaigns = ({ accessCampaigns, grantAccess, revokeAccess, state }) => {
+  return accessCampaigns.length > 0 && accessCampaigns.map((campaign, key) => {
```

Comment:
why is `accessCampaigns.length > 0` necessary? Returning `[]` or `false` from a component results in the same dom.

Reply:
It isn't.